services/web/server/src/simcore_service_webserver/version_control_db.py

```diff
@@ -129,49 +125,83 @@ async def _get_head_branch(
             )
             return branch
 
-    async def _get_head_commit(
+    async def _get_HEAD_commit(
         self, repo_id: int, conn: SAConnection
-    ) -> Optional[RowProxy]:
+    ) -> Optional[CommitProxy]:
```

Comment:
actually same as with ProjectProxy... I wonder. here ideally it should be some domain model going around right? not a aiopg specific object. but that is a minor.

Reply:
In this case, I disagree:
1. this is a protected function used as a wrapper around ``aiopg.sa`` interface, therefore outputs are ``*Proxy``-like data structures
2. This is an alias of ``RowProxy`` to clarify the table we are getting
3. i/o public methods of ``VersionControlRepository`` do have their own models (e.g. CommitLog) and yes, those know nothing about ``aiopg.sa`` models